wallet/keypass-json.js

```diff
@@ -0,0 +1,150 @@
+const scrypt = require('scryptsy')
```

Comment:
I would add some comment with link to the code which was used as a `template`

Reply:
I added a reference to the original github repo above, fixed